apps/emqx_bridge_kafka/test/emqx_bridge_kafka_impl_producer_SUITE.erl

```diff
@@ -416,7 +417,6 @@ kafka_bridge_rest_api_helper(Config) ->
         % this delete should not be necessary beause of the also_delete_dep_actions flag
         % {ok, 204, _} = http_delete(["rules", RuleId]),
         {ok, 204, _} = http_delete(BridgesPartsIdDeleteAlsoActions),
-        false = MyKafkaBridgeExists(),
```

Comment:
Nit: this assertion still seems to make sense here (just need to use `http_get_bridges`).